client.go

```diff
@@ -1989,19 +1990,35 @@ func (c *Client) GetMigrationSourceWS(container string) (*Response, error) {
 	return c.post(url, body, Async)
 }
 
-func (c *Client) MigrateFrom(name string, operation string, certificate string, secrets map[string]string, architecture string, config map[string]string, devices shared.Devices, profiles []string, baseImage string, ephemeral bool) (*Response, error) {
+func (c *Client) MigrateFrom(name string, operation string, certificate string,
```

Comment:
I don't know, I don't really think it would be any more invasive than the proposed change. No reason we can't make things a little bit better now :). Anyway, it's fine with me if it doesn't go in, but I don't see "there will be a rewrite soon" as a reason for it not to go in.